query.js

```diff
@@ -271,13 +271,33 @@ var searchMap = {
         ]
     }),
     type: d => ({ type: d }),
-    index: d => ({
-        $or: 
-        [ 
-            { index: { $in: d.startIndex.split(",").map(parseFloat) } },
-            { index: { $gte: parseFloat(d.startIndex), $lte: parseFloat(d.endIndex) } },
-        ]
-    }),
+    index: d => {
+        let commaSepValues = d.startIndex.split(",");
```

Comment:
Possible to refactor this out to a separate function and add a brief unit test for this, if not too much trouble? Like https://github.com/cestastanford/grandtour/blob/master/__tests__/query.unit.test.js#L11-L15 -- just to check a few cases.

And if people add spaces between hyphens / commas, will this also work?